hscli.cabal

```
@@ -75,6 +75,7 @@ common deps
     , vector                       ^>=0.12.0.3
     , yaml                         ^>=0.11.1
     , yarn-lock                    ^>=0.6.2
+    , xml
```

```suggestion
    , xml                          ^>=1.3.14
```
(adding the version bound)